src/main/java/movie/tickets/annotation/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,23 @@
+package movie.tickets.annotation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator implements ConstraintValidator<FieldsValueMatch, Object> {
+    private String field;
+    private String fieldMatch;
+
+    public void initialize(FieldsValueMatch constraint) {
+        this.field = constraint.field();
+        this.fieldMatch = constraint.fieldMatch();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext constraintValidatorContext) {
```

Comment:
```suggestion
    public boolean isValid(UserRequestDto value, ConstraintValidatorContext constraintValidatorContext) {
```
can we use UserRequestDto here?